cmd/open.go

```diff
@@ -16,5 +16,27 @@ func (h *Handler) Open(ctx context.Context, req *entity.CommandRequest) error {
 		return err
 	}
 
+	if (req.Cmd.Use != "open") {
+		return h.ctrl.OpenProjectPathInBrowser(ctx, projectId, environmentId, req.Cmd.Use)
+	}
+
 	return h.ctrl.OpenProjectInBrowser(ctx, projectId, environmentId)
 }
+
+func (h *Handler) OpenApp(ctx context.Context, req *entity.CommandRequest) error {
```

Comment:
Is there a better way to do this? I am fetching the latest deployment and then using the `staticUrl` from within it.

Reply:
I think that is the best way